pkg/localize/locales/en/cmd/kafka.en.toml

```diff
@@ -402,6 +402,13 @@ one = 'Unique ID of the Kafka instance you want to view'
 description = 'Description for the --name flag'
 one = 'Name of the Kafka instance you want to view'
 
+[kafka.describe.flag.bootstrapserver]
```

Comment:
In long run we want to have ability to get some specific config of the services in unified fassion. 

It is good to have this in describe for now, but we should have similar capability for the status command